build.sbt

```diff
@@ -98,6 +98,7 @@ lazy val core = libraryProject("core")
       log4s,
       parboiled,
       vault,
+      specs2ScalacheckTest(scalaVersion.value)
```

Comment:
Most of core is tested in the `tests` project, so it can depend on `testing`, which we also publish for third-party extensions.  That should already have this dependency, in addition to the `Arbitrary` that you had to reinvent.

We should make this clearer in the contributors guide.

Reply:
Ah, I see. Thanks, Ross. I believe that I've addressed your comments via https://github.com/http4s/http4s/pull/3167#commits-pushed-ff9cae6.